src/v3/src/components/AuthContainer/style.css

```diff
@@ -12,6 +12,6 @@
 /* TODO: this max width may chaged, will discuss w/ UX */
 @media screen and (max-width: 391px) {
   .siwContainer {
-    border: none;
+    border: none !important;
```

Comment:
any way to avoid the `!important`? given that we're already using css modules, it should be avoidable most of the time

Reply:
For some reason AuthContainer is not using css module.  Switching to css module breaks the styling for some reason.

Reply:
The `!important` is needed because we are setting the border as a prop in the MUI box which overrides this css property.  The alternative was to set the border to the .siwContainer class in the css file but that didn't seem like a very flexible option.

Reply:
I see. Can you try using https://mui.com/material-ui/react-use-media-query/ to match the same media query and toggle the border properties in JS instead? We'll be able to remove the media query block from the style file this way.

Reply:
Cool I was able to use the mui use media query and it works so I removed the media query block from the css.